src/index.ts

```diff
@@ -1,11 +1,6 @@
 import * as semver from 'semver';
 import memoize = require('lodash.memoize');
 
-// TODO: PR to DefinitelyTyped project
-declare module 'semver' {
```

Comment:
I guess that this was somehow lost since travis build is failing (`Property 'parse' does not exist on type 'typeof "/home/travis/build/resin-io-modules/resin-semver/node_modules/@types/semver/index"`).
Was this supposed to move in a `.d.ts` file or are we waiting for a DT PR?